src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.scss

```diff
@@ -7,5 +7,10 @@
 }
 
 .color-neutral {
-  color: gray;
+  color: green;
+  font-weight: bold;
+}
```

Comment:
Ideally, font-weight shouldn't be modified in a utility class named `color-neutral`. Suggest we see if it's possible to use the color positive class in addition to bootstrap's `font-weight-bold`.